server/config/mongoDB.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+const config = require('./config')
+
+mongoose.connect(config.mongoUri, (err) => {
+    if (err) {
+        throw err
+    } else {
+        console.log("mongodb connected:D")
+    }
+})
+
+module.exports = mongoose
```

Comment:
We don't need to import this file everywhere. We can directly import mongoose. Lets use this just where we are initialising express and don't export it.

Reply:
i did this because the will be modular ,this is only imported in schema

Reply:
Well, it's already modular. You just connect mongoose and import mongoose directly in the schemas to define them. They don't need to depend on mongoose connection.